### PR TITLE
Added documentation for general introduction to APBs and ASB

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+The approach discussed has focused on the methods defined by the Open Service Broker API, yet we are envisioning
+additional methods to be added in the future to assist with cases of upgrading, downgrading, testing, verifying, etc.
+
+We are targeting a working demonstration for Red Hat Summit with a deliverable of the Ansible Service Broker in a summer release.
+
+To keep up with developments or to learn more:
+ * [Trello](https://trello.com/b/50JhiC5v/ansible-apps)
+ * Github
+   * [Ansible Playbook Bundle Design Documents](https://github.com/fusor/ansibleplaybookbundle/tree/master/docs)
```
Comment (jmrodri) [inline docs/introduction.md]: Change url to one that works: https://github.com/fusor/ansible-playbook-bundle/tree/master/docs

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+  3.) Ansible Playbook Bundle workflow
+
+    a.) Preparing an Ansible Playbook Bundle
+
+    b.) Packaging an Ansible Playbook Bundle
+
+    c.) Deploying an Ansible Playbook Bundle
+
+### Ansible Service Broker
```
Comment (jmrodri) [inline docs/introduction.md]: Why not make the ### headings ## then make the bolded ones ###.

File: docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+for OpenShift to manage Ansible Playbook Bundles.
+
+Users will interact with the [Service Catalog](https://github.com/kubernetes-incubator/service-catalog) to obtain a list of available applications and invoke operations
+for provisioning, deprovisioning, binding, and unbinding to an application. The Service Catalog will rely on
+a collection of brokers to handle details associated with its applications. The diagram below illustrates a
+high level concept of the workflow.
+
+![Overview](ansible-service-broker-overview.png)
+
+**Service Catalog to Service Broker Workflow**
```
Comment (jmrodri) [inline docs/introduction.md]: Why not make the ### headings ## then make the bolded ones ###.  Make this one ###

File: docs/introduction.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  1.) User requests list of available applications from the Service Catalog
+
+  2.) Service Catalog asks the Ansible Service Broker for available applications
+
+  3.) Ansible Service Broker talks to a container registry to learn which APBs are available
+
+  4.) User issues a command to provision a specific APB
+
+  5.) Provision request makes its way to the Ansible Service Broker which fulfills the command by invoking the
+      provision method on the APB
```
Comment (jmrodri) [inline docs/introduction.md]: I would reformat this to be single spaced, then just use the 1,2,3--- button. i.e. 

```
1. User requests list of available applications from the Service Catalog
2. Service Catalog asks the Ansible Service Broker for available applicatins
```

File: docs/introduction.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+Specification of an APB is intended to be lightweight, consisting of several named playbooks and a
+metadata file to capture information such as parameters to pass into the application.
+
+The workflow for an APB is broken up into three steps:
+
+  1.) Prepare
+
+  2.) Package
+
+  3.) Deploy
```
Comment (jmrodri) [inline docs/introduction.md]: Same comment about above numbered list. Probably applies to any other ones in the document.

**Comment by jwmatthews** [comment]
ACK

**Pull request merged**